float64 x
float64 y
float64 z
char[2] navStat
float64 hAcc
float64 vAcc
float64 SOG
float64 COG
float64 vel
float64 HDOP
float64 VDOP
float64 TDOP
int64 numSat


